features/step_definitions/gallery_steps.rb

```diff
@@ -0,0 +1,11 @@
+Then 'I should see the photo lightbox' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.